pkg/cmd/server/api/v1/types.go

```diff
@@ -213,6 +213,9 @@ type NetworkConfig struct {
 	ClusterNetworkCIDR string `json:"clusterNetworkCIDR"`
 	HostSubnetLength   uint   `json:"hostSubnetLength"`
 	ServiceNetworkCIDR string `json:"serviceNetworkCIDR"`
+
+	// AdminNamespaces are the namespaces/projects that are allowed to reach all networks in case of multi-tenant SDN
+	AdminNamespaces []string `json:"adminNamespaces"`
```

Comment:
@liggitt  How about privileged namespaces instead of calling admin namespaces? Privileged namespaces will get VNID=0 and they can access any pods in the cluster.